index.d.ts

```diff
@@ -0,0 +1,59 @@
+declare namespace DbTypes {
+    type ClientType = 'mysql' | 'sqlite' | 'postgres';
+
+    interface DbErrorOpts {
+        nativeError: {
+            message: string
+        }
+        client: ClientType
+    }
+
+    class DBError extends Error {
+        name: string;
+        nativeError: Error;
+        client: ClientType;
```

Comment:
I don't think `client` is useful information. Maybe it should be removed? Or do you have a use case in mind? This effectively makes this property part of the public API and we can never rename or remove it.

Reply:
Makes sense, removed